### DISCUSSION TITLE
Single PC - docker, multi HDD, erasure

### DISCUSSION THREAD

**Discussion by eleaner** [seaweedfs/seaweedfs#3411] [General]
Hey,

How would seaweed work in docker deployment on a single machine with multiple HDD and erasure to secure HDD failure?

Does anyone have experience with that?

**Comment by chrislusf** [comment]
you can run one volume server on each HDD and use replication.

**Comment by eleaner** [comment]
Yup, the docker setup is pretty easy.
I marked each HDD as different rack and set the replication to 020
How many HDD I would need for EC to kick in? 14+ ?

**Reply by Stitch10925** [reply]
For EC you need at least 4 nodes:

https://github.com/seaweedfs/seaweedfs/discussions/2899
